parlai/mturk/core/react_server/dev/app.jsx

```diff
@@ -54,16 +54,19 @@ function getHitConfig(callback_function) {
   });
 }
 
-// Sees if the current browser supports WebSockets
+// Sees if the current browser supports WebSockets, using *bowser*
+/*eslint-disable */
```

Comment:
Ah nice catch, I had thought that was an earnest typo. Thanks for the comment updates to clarify.

Cool, so this should do the trick. Another option (slightly preferable) would be to do: `/* global bowser */` instead (see: https://eslint.org/docs/user-guide/configuring#specifying-globals).
This is a little bit stricter since it only whitelists this specific rule-bypass, and will still catch other potential errors (although unlikely since you're only disabling eslint for a very small piece of the code).

Reply:
No worries, I should've caught this as I've actually broken this very thing before in the past. Great context also - I'll use the global formatting instead as it's certainly more accurate.